externals/sha3/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+project(SHA3)
+
+add_library(cvmfs_sha3 STATIC
+  src/64opt/KeccakF-1600-opt64.c
+  src/KeccakHash.c
+  src/KeccakSponge.c
+  src/SnP-FBWL-default.c
+  )
+
+set_property(TARGET cvmfs_sha3 PROPERTY POSITION_INDEPENDENT_CODE TRUE)
+
+target_include_directories(cvmfs_sha3 PUBLIC
+  ${CMAKE_CURRENT_SOURCE_DIR}/src
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/64opt
+  )
+
+set(SHA3_INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/src ${CMAKE_CURRENT_SOURCE_DIR}/src/64opt PARENT_SCOPE)
+set(SHA3_LIBRARY $<TARGET_FILE:cvmfs_sha3> PARENT_SCOPE)
```

Comment:
Is the library stripped?

Reply:
That will probably depend on the configuration. Let me know if they should/should not be stripped by force and I can make sure that this will be the case.

Reply:
Yes, please enforce stripping the library (we don't expect any meaningful stack trace information in this library)

Reply:
I guess setting visibility of symbols to hidden achieves what you want, right?
See https://cmake.org/cmake/help/latest/prop_tgt/LANG_VISIBILITY_PRESET.html.

Reply:
Or you mean that we should strip debugging symbols?